gwswlib/tests/test_importer.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+"""Tests for importer.py"""
+from gwswlib.importer import check_headers
+
+
+def test_check_headers(caplog):
+    a = [1, 2, 3]
+    b = [2, 3, 4]
+    check_headers(a, b)
+    assert "missing columns" in caplog.text
+    assert "extra columns" in caplog.text
```

Comment:
In dit geval prima om twee asserts te hebben.

Maar *in het algemeen* wil je maar één check per testfunctie hebben. Want hij stopt na de eerste falende check. En ik heb het al meerdere keren gehad dat ik dacht dat iets helemaal mis ging terwijl het alleen de eerste check was. Als de tweede en derde ook gedraaid hadden had ik de info gehad die ik nodig had.